.github/workflows/e2e-matrix.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        suite: [Nonbehavioral, Utilization, GPU, Drift, Integration, NodeClaim]
+        suite: [Nonbehavioral, Utilization, GPU, Drift, Integration, NodeClaim, Arm]
```

Comment:
ARM Is selected almost every time due to pricing for most of these, so multi-arch introduces some complications where we don't test the traditional arch often enough. I would prefer to have much larger coverage for amd64. 

This is why I added a separate suite for arm as opposed to that, let me know your thoughts

Reply:
The majority of tests apply to both - especially with multi-arch workloads/images - which would point in the direction of architecture being yet another parameter in the test matrix. We don't have to let architecture be picked up automatically; can eliminate the non-determinism or external factors affecting the choice of architecture in test by specifying it explicitly. There will still be tests targeting specific architecture, or maybe custom multi-arch scenarios, but that would be a small set.

Reply:
Yes thats true, do we want to add additional matrix cases to each of the test suites then(Similar to what I did for azure linux utilization?) ? We should keep this test the way it is until we have parameterization, rather than doubling the time it takes to run our full suite. 

Thoughts?

Reply:
Ok for now i am just going to run as DescribeTable style ginkgo test runnign all the permutations we want. I would prefer we eventually convert to launching new clusters for each parameter.